src/USER-MDI/mdi_engine.cpp

```diff
@@ -0,0 +1,364 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Taylor Barnes (MolSSI)
+   MolSSI Driver Interface (MDI) support for LAMMPS
+------------------------------------------------------------------------- */
+
+#include <mpi.h>
```

Comment:
please see https://github.com/lammps/lammps/blob/master/doc/include-file-conventions.md for current conventions and suggestions for include files. This file looks like it is based on a file from quite a while ago that and thus is missing several iterations of the include file cleanups we did.